### DISCUSSION TITLE
insert nulos inteiros

### DISCUSSION THREAD

**Discussion by PauloCotrim** [fernandobatels/rsfbclient#130] [General]
Desculpe pela ignorancia, sei que o rust não tem nulos.

Existe alguma forma de inserir nulo em um campo inteiro ? 

Eu faço os parametros na struct e defino os tipos.

**Comment by fernandobatels** [comment]
O null é tratado com [Option](https://doc.rust-lang.org/std/option/enum.Option.html).

Sua struct ficaria parecida com isso:
``` rust
#[derive(Clone, IntoParams)]
struct ParamTest {
   meu_campo: Option<i32>
}
```

**Comment by fernandobatels** [comment]
Temos um exemplo nos testes: https://github.com/fernandobatels/rsfbclient/blob/8bf69e5764e365ea27bb13bc418506e255ddfe68/src/tests/params.rs#L42

**Reply by PauloCotrim** [reply]
Muito Obrigado.
